cmd/utils/flags.go

```diff
@@ -893,6 +893,11 @@ var (
 		Usage:    "By default the pending block equals the latest block to save resources and not leak txs from the tx-pool, this flag enables computing of the pending block from the tx-pool instead.",
 		Category: flags.RollupCategory,
 	}
+	RollupHaltOnIncompatibleProtocolVersionFlag = &cli.StringFlag{
```

Comment:
This doesn't seem to have been added to the list of flags in `geth/main.go` to make it actually available.